### DISCUSSION TITLE
This is a question for the contributors of sonic worlds next

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#98] [Q&A] [answered]
Do any of you have a discord accout if yes join this server link

SonicWorldsNext discord server link
[Link removed]

**Comment by DimensionWarped** [comment, marked as answer]
If and when we get a Discord server for this project, it will be under the stewardship of @Techokami or someone he directly appoints to take on that task.

**Comment by Renhoex** [comment]
Adding to DimensionWarped's post, we've floated the idea a few times but we keep deciding against it as we want all engine support questions to be on the git page so that people don't have to join a server and repeat the same question someone else already asked

**Comment by Techokami** [comment]
I have removed the links because it is not an official Discord server for the project. And please, if you do not stop spamming threads on the discussion forums here, I will have no choice but to remove you as well.
